src/webgpu/api/validation/attachment_compatibility.spec.ts

```diff
@@ -206,8 +224,14 @@ g.test('render_pass_and_bundle,depth_format')
       .combine(poptions('passFormat', kDepthStencilAttachmentFormats))
       .combine(poptions('bundleFormat', kDepthStencilAttachmentFormats))
   )
-  .fn(t => {
+  .fn(async t => {
     const { passFormat, bundleFormat } = t.params;
+
+    const extensions = t.getTextureFormatsRequiredExtensions([passFormat, bundleFormat]);
```

Comment:
See the comment above. It is hard to generalize this function, it will report type mismatch because the types might be:
UnsizedDepthStencilFormat,
DepthStencilFormat,
UncompressedTextureFormat,
AllTextureFormat / GPUTextureFormat

But it requires strict match.

Reply:
> The resulting helper would be generally useful (it could be extended to all texture formats) and could live in GPUTest. (In which case please document it in `helper_index`.) It should work for all of the cases in the rest of this PR I think.

I think this is the only outstanding comment

Reply:
Done. I also documented another helper function I added a few days ago.